src/features/ITGovernance/Admin/GRBReview/GRBReviewForm/GRBReviewFormStepWrapper/index.tsx

```diff
@@ -282,12 +253,7 @@ function GRBReviewFormStepWrapper<
 
           <Pager
             next={{
-              type: onSubmit ? 'submit' : 'button',
-              onClick: () =>
-                !onSubmit &&
-                history.push(`${grbReviewPath}/${nextStep?.key || ''}`),
-              // Disable `next` button if next step is disabled
-              disabled: !!nextStep?.disabled,
+              disabled: !isValid || !!nextStep?.disabled,
```

Comment:
This will always be disabled until the data is saved by autosave, as noted by above.  I'n my branch I had an alternative true condition that relied solely on the the current form validation, rather the validation of the existing/persisted data